website/layouts/partials/_default/version-picker.html

```diff
@@ -0,0 +1,8 @@
+<div class="dropdown show">
+    <a class="nav-link dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Version</a>
```

Comment:
is this used?

Reply:
let's remove

Reply:
Done. Removed vpick instead and copied over the code into this file